v2/call_option.go

```diff
@@ -119,6 +121,38 @@ func (r *boRetryer) Retry(err error) (time.Duration, bool) {
 	return 0, false
 }
 
+// OnHTTPCodes returns a Retryer that retries if and only if
+// the previous attempt returns a googleapi.Error whose status code is stored in
+// cc. Pause times between retries are specified by bo.
+//
+// bo is only used for its parameters; each Retryer has its own copy.
+func OnHTTPCodes(cc []int, bo Backoff) Retryer {
+	return &httpRetryer{
+		backoff: bo,
+		codes:   append([]int(nil), cc...),
```

Comment:
Consider transforming this into a set `map[string]bool` to avoid future iterations.

Reply:
Yeah I'm down with that.